scripts/signedbuildskadi.sh

```diff
@@ -24,6 +24,7 @@ if [ $default_skadi_passwords = "false" ]
     TIMESKETCH_PASSWORD=$(openssl rand -base64 32 |sha256sum | sed 's/ //g')
     NGINX_USER="skadi_$(openssl rand -base64 3)"
     NGINX_PASSWORD=$(openssl rand -base64 32 |sha256sum | sed 's/ //g')
+    HEAP_SIZE=$HEAP_SIZE
```

Comment:
I think this will need to be passed as an env variable with a default option if no env variable found.

for example
HEAP_SIZE=${HEAP_SIZE:-1g)

Reply:
should be corrected in lastest commit